pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -22,6 +22,20 @@ package pulsar.proto;
 option java_package = "org.apache.pulsar.common.api.proto";
 option optimize_for = LITE_RUNTIME;
 
+message Schema {
+	required string name = 1;
+	required bytes version = 2;
+	required bytes schema_data = 7;
```

Comment:
We should probably use 3 & 4 here to avoid implying we have retired some fields

Reply:
Yeah definitely, I deleted a few fields and didn't readjust the numbering.